docs/markdown/httpapi/api_spec.md

```diff
@@ -710,6 +710,13 @@ The statistic entries are dependent on the daemon type.
 Values are returned as strings.
 
 
+Authoritative: the statistics are:
+
+* with type = "StatisticItem": [Standard Performance Counters](../authoritative/performance.md#counters-variables)
+* with type = "QueryTypeStatisticItem": per-qtype query counters (same as `pdns_control qtypes`)
```

Comment:
Awesome the pdns_control docs (in authoritative/internals.md) are missing the `qtypes`. Could you add those and link there? Or shall I fix that after this is merged?